### PR TITLE
Add caption icons

### PR DESCRIPTION
**Pull request by DefaultLP**
## What is the purpose of this pull request?

- [x] New Icon
- [ ] Bug fix
- [ ] New Feature
- [ ] Documentation update
- [ ] Other:

### Description
This PR adds two new icons. One for captions and one for disabled captions.
(GitHub did not let me patch the last pull request for some reason... so now with actually 2px corner radius...)

inkscape_jHkPsnRJ0D

Additionally, this PR adds the "multimedia" category to the subtitles icon. Same as these captions.

### Icon use case
Is it useful for any application that deals with videos. Whether that be streaming or editing, having an icon that represents one of the three major components of internet videos is always a good idea. 

I mainly did this because I needed a subtitles/captions icon that had an off/disabled variant since I'm building a SvelteKit UI that works with videos.

### Alternative icon designs

Off/disabled variant
inkscape_ShBHixQJuk

### Concept
- [x] I have provided valid use cases for each icon.
- [x] I have not added any a brand or logo icon.
- [x] I have not used any hate symbols.
- [x] I have not included any religious or political imagery.

### Author, credits & license
- [ ] The icons are solely my own creation.
- [ ] The icons were originally created in #<issueNumber> by @<githubUser>
- [x] I've based them on the following Lucide icons: subtitles
- [ ] I've based them on the following design: 

### Naming
- [x] I've read and followed the [naming conventions](https://lucide.dev/guide/design/icon-design-guide#naming-conventions)
- [x] I've named icons by what they are rather than their use case.
- [x] I've provided meta JSON files in `icons/[iconName].json`.

### Design
- [x] I've read and followed the [icon design guidelines](https://lucide.dev/guide/design/icon-design-guide)
- [x] I've made sure that the icons look sharp on low DPI displays.
- [x] I've made sure that the icons look consistent with the icon set in size, optical volume and density.
- [x] I've made sure that the icons are visually centered.
- [x] I've correctly optimized all icons to two points of precision.

## Before Submitting 

- [x] I've read the [Contribution Guidelines](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).
- [x] I've checked if there was an existing PR that solves the same issue.

### CODE REVIEW

File: icons/captions-off.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "defaultlp"
+  ],
+  "tags": [
+    "captions",
+	"closed captions",
+    "subtitles",
+	"accassibility"
+  ],
```
Comment (ericfennis) [inline icons/captions-off.json:11]:
```suggestion
    "DefaultLP"
  ],
  "tags": [
    "closed captions",
	  "subtitles",
	  "subhead",
	  "transcription",
	  "transcribe",
	  "dialogue",
    "accessibility"
  ],
```

File: icons/captions.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "defaultlp"
+  ],
+  "tags": [
+    "captions",
+	"closed captions",
+    "subtitles",
+	"accassibility"
+  ],
```
Comment (ericfennis) [inline icons/captions.json:11]:
```suggestion
  "contributors": [
    "DefaultLP"
  ],
  "tags": [
     "closed captions",
	  "subtitles",
	  "subhead",
	  "transcription",
	  "transcribe",
	  "dialogue",
    "accessibility"
  ],
```

File: icons/captions.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-captions"><rect width="18" height="14" x="3" y="5" rx="2" ry="2"/><path d="M7 15h4M15 15h2M7 11h2M13 11h4"/></svg>
```
Comment (ericfennis) [inline icons/captions.svg:1]: Can you make sure the formatting is correctly done?

File: icons/captions-off.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-captions-off"><path d="M7 15h4M7 11h2M2 2l20 20"/><path d="M17 11h-1M19 19H5s-2 0-2-2V7s-.28-2 2-2M21 16V7s0-2-2-2h-9"/></svg>
```
Comment (ericfennis) [inline icons/captions-off.svg:1]: This file need some formatting, see other icons as example.
Or see the lining errors to guide you.

**Review by ericfennis — CHANGES_REQUESTED** [review]
Nice icon, the design is perfect.
The SVG code needs some formatting. And the meta json files as well, I gave some suggestions.

I think we should replace the current `subtitles` icon, with these ones.

**Comment by DefaultLP** [comment]
I now changed all files to reflect the feedback.

Although the SVG formatting was an incredibly stupid error on my part, since I used the SVG I downloaded from the icons library as reference, instead of the ones from the repository...

**Comment by DefaultLP** [comment]
So sorry for that. I'm usually not such a pain to work with

**Comment by jguddas** [comment]
The off variant has a bunch of gap violations and should use arcs for the corners.

This would work icon (36)

https://lucide-studio.vercel.app/?value=%3Csvg++xmlns=%22http://www.w3.org/2000/svg%22++width=%2224%22++height=%2224%22++viewBox=%220+0+24+24%22++fill=%22none%22++stroke=%22currentColor%22++stroke-width=%222%22++stroke-linecap=%22round%22++stroke-linejoin=%22round%22%3E++%3Cpath+d=%22M10.5+5H19a2+2+0+0+1+2+2v8.5%22+/%3E++%3Cpath+d=%22M17+11h-.5%22+/%3E++%3Cpath+d=%22M19+19H5a2+2+0+0+1-2-2V7a2+2+0+0+1+2-2%22+/%3E++%3Cpath+d=%22m2+2+20+20%22+/%3E++%3Cpath+d=%22M7+11h4%22+/%3E++%3Cpath+d=%22M7+15h2.5%22+/%3E%3C/svg%3E

**Comment by DefaultLP** [comment]
Well and now I know what gap violations are. What a productive day.

File: icons/captions-off.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "defaultlp"
```
Comment (ericfennis) [inline icons/captions-off.json:4]: Your name should be exactly the same as in you Github URL like. So it should be:
```suggestion
    "DefaultLP"
```

File: icons/captions.json [hunk 6]
```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "defaultlp"
```
Comment (ericfennis) [inline icons/captions.json:4]: Same here:
```suggestion
    "DefaultLP"
```

File: icons/captions.json [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+	  "subtitles",
+	  "subhead",
+	  "transcription",
+	  "transcribe",
+	  "dialogue",
+    "accessibility"
+  ],
+  "categories": [
+    "multimedia"
+  ]
```
Comment (ericfennis) [inline icons/captions.json:17]: We should remove the `subtitles` icon and add it as an alias.
```suggestion
  ],
  "aliases": [
    "subtitles"
  ]
```

File: icons/subtitles.json [hunk 8]
```diff
@@ -9,5 +9,13 @@
     "closed captions",
     "accessibility"
```
Comment (ericfennis) [inline icons/subtitles.json:10]: Can you remove this file along with the SVG file?

**Review by ericfennis — CHANGES_REQUESTED** [review]

**Comment by jguddas** [comment]
My vote, keep them both and just rename the `subtitels` icon to `message-square-text`.

**Comment by jguddas** [comment]
We can add the alias there.

**Review by ericfennis — APPROVED** [review]
Nice work @DefaultLP !

**Comment by DefaultLP** [comment]
Well, thank you for putting up with me!

**Pull request merged**